packages/web-pkg/src/composables/sse/useServerSentEvents.ts

```diff
@@ -1,79 +0,0 @@
-import { EventSourceMessage, fetchEventSource } from '@microsoft/fetch-event-source'
-import { ref, unref, watch } from 'vue'
-import { v4 as uuidV4 } from 'uuid'
-import { useGettext } from 'vue3-gettext'
-import { configurationManager } from '../../configuration'
-import { useAccessToken } from '../authContext'
-import { useStore } from '../store'
-
-class FatalError extends Error {}
-
-export interface ServerSentEventsOptions {
-  url: string
-  onOpen?: (response: Response) => void
-  onMessage?: (msg: EventSourceMessage) => void
-}
-
-export const useServerSentEvents = (options: ServerSentEventsOptions) => {
-  const store = useStore()
-  const language = useGettext()
-  const accessToken = useAccessToken({ store })
-  const ctrl = ref(new AbortController())
-  const maxRetries = 3
-  const retryCounter = ref(0)
-
-  watch(
-    () => language.current,
```

Comment:
Reconnecting with the updated language has been lost in this PR, no?

Reply:
Yep, I think we can do this as well. Will be another external call but ok for now.